DfciPkg/SettingsManager/SettingsManagerProvider.c

```diff
@@ -332,11 +332,10 @@ SetProviderValueFromAscii (
   return Status;
 }
 
-#define ENABLED_STRING_SIZE                (9)
-#define ASSET_TAG_STRING_MAX_SIZE          (22)
+#define ENABLED_STRING_SIZE                (13)

```

Comment:
I'm not familiar with the strings but is there a more reliable way to calculate this?

Reply:
"Inconsistent" was added in commit 5ab12d9aba584400e9edcd3cf11be8a348f997d5, but the size was not updated.

Reply:
It would be better to allocate the buffer with the right size of each string. But this needs to refactor the function, introducing more changes.